content/en/docs/release-notes/release-notes-1.7.md

```diff
@@ -7,12 +7,117 @@ type: "docs"
 
 ## Breaking Changes (You **MUST** read this before you upgrade!)
 
+⚠ Following their deprecation in version 1.5, the cert-manager API versions v1alpha2, v1alpha3, and v1beta1 have been removed.
+You must ensure that all cert-manager custom resources are stored in etcd at version v1
+and that all cert-manager `CustomResourceDefinition`s have only v1 as the stored version.
+
+Since release 1.7, `cmctl` can automatically migrate any deprecated API resources.
+Please download `cmctl-v1.7.0-beta.0` and read [Removing Deprecated API Resources]
+for full instructions.
+
+[Removing Deprecated API Resources]: https://cert-manager.io/docs/installation/upgrading/remove-deprecated-apis/
+
 ## Major Themes
 
+### Additional Certificate Output Formats
+
+`additionalOutputFormats` is a field on the Certificate `spec` that allows
+specifying additional supplementary formats of issued certificates and their
+private key. There are currently two supported additional output formats:
+`CombinedPEM` and `DER`. Both output formats can be specified on the same
+Certificate.
```

Comment:
suggestion:

```suggestion
private key. There are currently two supported additional output formats:
`CombinedPEM` (the PEM-encoded private key followed by the certificate chain)
and `DER` (the DER-encoded private key only). Any combination of output formats
can be requested for the same certificate.
```

Reply:
I should make this same change in `usage/certificate/#additional-certificate-output-formats`